### PR TITLE
Live attenuated-vaccines

### PR DESCRIPTION
**Pull request by RLordan**
Hey @rando2 apologies for the delay. Chop and change as you see fit.

### Suggested reviewers (optional)
 @rando2 @agitter

### CODE REVIEW

File: content/22.vaccines.md [hunk 1]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+The virus can be attenuated by passinging it in a foreign host, until as a consequence of selection pressure the virus loses its efficacy in the original host. 
+Alternatively, selective gene deletion or codon de-optimization can be utilized to attenuate the virus [@doi:10.1038/s41586-020-2798-3]. 
+LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
+It is generally recognized that LAVs induce an immune response that is similar to natural infection and are favoured because they induce long-lasting and robust immunity that can protect from disease. 
+This is in part as a result of the immune response to the range of viral antigens available that tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256].
```
Comment (rando2) [inline content/22.vaccines.md:131]: I found this sentence a little hard to follow -- would these edits keep the same meaning?
```suggestion
This strong protective effect is induced in part by the immune response to the range of viral antigens available from LAV, which tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256]. 
```

File: content/22.vaccines.md [hunk 2]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+Alternatively, selective gene deletion or codon de-optimization can be utilized to attenuate the virus [@doi:10.1038/s41586-020-2798-3]. 
+LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
+It is generally recognized that LAVs induce an immune response that is similar to natural infection and are favoured because they induce long-lasting and robust immunity that can protect from disease. 
+This is in part as a result of the immune response to the range of viral antigens available that tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256]. 
+LAVs are also favoured because they tend to be restricted to viral replication in the tissues around the location of inoculation [@doi:10.1128/9781555818951] and some can be administered intranasally [@doi:10.1038/s41586-020-2798-3].
```
Comment (rando2) [inline content/22.vaccines.md:132]:
```suggestion
LAVs are also favored because they tend to be restricted to viral replication in the tissues around the location of inoculation [@doi:10.1128/9781555818951], and some can be administered intranasally [@doi:10.1038/s41586-020-2798-3]. 
```
I feel bad about the edits that are just for 🇺🇸 spelling!

File: content/22.vaccines.md [hunk 3]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
+It is generally recognized that LAVs induce an immune response that is similar to natural infection and are favoured because they induce long-lasting and robust immunity that can protect from disease. 
+This is in part as a result of the immune response to the range of viral antigens available that tend to be more immunogenic than those from non-replicating vaccines [@url:https://www.hhs.gov/immunization/basics/types/index.html; @doi:10.1016/j.virol.2015.03.032; @doi:10.3389/fimmu.2020.602256]. 
+LAVs are also favoured because they tend to be restricted to viral replication in the tissues around the location of inoculation [@doi:10.1128/9781555818951] and some can be administered intranasally [@doi:10.1038/s41586-020-2798-3]. 
+
+The first deliberate attempt to utilize an attenuated viral vaccine dates back to Louis Pasteur in 1885, despite not knowing that the disease-causing agent he was experimenting with was a virus.
```
Comment (rando2) [inline content/22.vaccines.md:134]:
```suggestion
The first deliberate attempt to utilize an attenuated viral vaccine dates back to Louis Pasteur in 1885, despite his not knowing that the disease-causing agent he was experimenting with was a virus. 
```

File: content/22.vaccines.md [hunk 4]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+LAVs are also favoured because they tend to be restricted to viral replication in the tissues around the location of inoculation [@doi:10.1128/9781555818951] and some can be administered intranasally [@doi:10.1038/s41586-020-2798-3]. 
+
+The first deliberate attempt to utilize an attenuated viral vaccine dates back to Louis Pasteur in 1885, despite not knowing that the disease-causing agent he was experimenting with was a virus. 
+Indeed, the next intentional LAVs developed were intended to prevent illness caused by the yellow fever virus in 1935, followed by the first influenza vaccine in 1936 [@doi:10.1128/9781555818951]. 
+Although LAV development strategies have the longest history, this strategy has not been widely utilized against SARS-CoV-2 and COVID-19. 
+There is general concern that LAV strategies may risk causing disease in individuals who are immunocompromised [@doi:10.1016/j.copbio.2007.10.010], which is an even greater concern when dealing with a novel virus and disease. 
+Previously, there have been numerous attempts to develop both SARS-CoV and MERS-CoV LAVs [@doi:10.3389/fimmu.2020.602256], but no vaccine has been approved for use in humans.
```
Comment (rando2) [inline content/22.vaccines.md:138]: The way it's written makes me think it's more like mRNA vaccines, where there were efforts but no history of approval. In this case, it's more just that these specific efforts didn't pan out. Does this work?
```suggestion
Previously, there have been numerous attempts to develop both SARS-CoV-1 and MERS-CoV LAVs [@doi:10.3389/fimmu.2020.602256], but no vaccines were approved for use in humans.
```

File: content/22.vaccines.md [hunk 5]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+Nowadays, the manufacture of LAVs use safe and reliable methods to produce large quantities of vaccines once they have undergone rigorous pre-clinical studies and clinical trials to evaluate their safety and efficacy.
```
Comment (rando2) [inline content/22.vaccines.md:139]: There is a mention of the polio outbreak in the intro, so I think maybe add a clause to hearken back to that?
```suggestion
While safety in production was a major concern in the past, nowadays manufacturers of LAVs use safe and reliable methods to produce large quantities of vaccines once they have undergone rigorous preclinical studies and clinical trials to evaluate their safety and efficacy.
```

File: content/22.vaccines.md [hunk 6]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+This LAV has been assessed in hamsters, mice, and macaques [@doi:10.1038/s41586-020-3035-9].
+YF-SO induced a robust immune response in all three animal models and prevented SARS-CoV-2 infection in macaques and hamsters [@doi:10.1038/s41586-020-3035-9].
+Other LAVs being investigated for the prophylaxis of COVID-19 include a Bacillus Calmette-Guerin (BCG) vaccine sponsored by institutes in Australia in collaboration with the Bill and Melinda Gates Foundation, which is in phase III clinical testing [@clinicaltrials:NCT04327206]. 
+A second investigation using a BCG vaccine is also ongoing, which is led by Texas A&M University in collaboration with numerous other U.S. institutions [@clinicaltrials:NCT04348370].
+The purpose of the BCG vaccine is to prevent tuberculosis, however, it is known to exert protective non-specific effects against other respiratory tract infections in _in vitro_ and _in vivo_ studies [@doi:10.1016/j.cmi.2019.04.020], hence the interest many have for its potential use against COVID-19 [@doi:10.1038/s41577-020-0337-y].
```
Comment (rando2) [inline content/22.vaccines.md:147]:
```suggestion
The purpose of the BCG vaccine is to prevent tuberculosis; however, it is known to exert protective non-specific effects against other respiratory tract infections in _in vitro_ and _in vivo_ studies [@doi:10.1016/j.cmi.2019.04.020], hence the interest many have for its potential use against COVID-19 [@doi:10.1038/s41577-020-0337-y].
```

File: content/22.vaccines.md [hunk 7]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+
+In april 2020, it was announced that the Indian Immunologicals Ltd. and Griffith University Australia had partnered to develop codon de-optimized LAV [@url:https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine/], however, there has been no updates on the findings of their preclinical testing.
```
Comment (rando2) [inline content/22.vaccines.md:149]:
```suggestion
In April 2020, it was announced that the Indian Immunologicals Ltd. and Griffith University Australia had partnered to develop codon de-optimized LAV [@url:https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine]; however, there have been no updates on the findings of their preclinical testing. 
```

File: content/22.vaccines.md [hunk 8]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+A second investigation using a BCG vaccine is also ongoing, which is led by Texas A&M University in collaboration with numerous other U.S. institutions [@clinicaltrials:NCT04348370].
+The purpose of the BCG vaccine is to prevent tuberculosis, however, it is known to exert protective non-specific effects against other respiratory tract infections in _in vitro_ and _in vivo_ studies [@doi:10.1016/j.cmi.2019.04.020], hence the interest many have for its potential use against COVID-19 [@doi:10.1038/s41577-020-0337-y].
+
+In april 2020, it was announced that the Indian Immunologicals Ltd. and Griffith University Australia had partnered to develop codon de-optimized LAV [@url:https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine/], however, there has been no updates on the findings of their preclinical testing. 
+Another codon de-optimized LAV is being developed by Mehmet Ali Aydinlar University and Acibadem Labmed Health Services A.S., which also has yet to report the findings of its preclinical tests [@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].
```
Comment (rando2) [inline content/22.vaccines.md:150]:
```suggestion
Another codon de-optimized LAV is being developed by Mehmet Ali Aydinlar University and Acibadem Labmed Health Services A.S., which also has yet to report the findings of its preclinical tests [@url:https://covid-19tracker.milkeninstitute.org].
```
Manubot gets mad about # but I was able to find this info easily just from the root URL!

Do we want to mention this is Turkey since we talk about the geography later on?

File: content/22.vaccines.md [hunk 9]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+Following successful preclinical investigation [@doi:10.1073/pnas.2102775118], an intranasally administered deoptimized SARS-CoV-2 LAV known as COVI-VAC was developed by both New York-based Codagenix and the Serum Institute of India. 
+COVI-VAC entered phase I human trials and dosed its first participants in January 2021 [04619628][@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].
```
Comment (rando2) [inline content/22.vaccines.md:152]:
```suggestion
COVI-VAC entered phase I human trials and dosed its first participants in January 2021 [@clinicaltrials:NCT04619628; @url:https://covid-19tracker.milkeninstitute.org].  
```

File: content/22.vaccines.md [hunk 10]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+
+In april 2020, it was announced that the Indian Immunologicals Ltd. and Griffith University Australia had partnered to develop codon de-optimized LAV [@url:https://news.griffith.edu.au/2020/04/23/griffith-university-researchers-on-the-road-to-covid-19-vaccine/], however, there has been no updates on the findings of their preclinical testing. 
+Another codon de-optimized LAV is being developed by Mehmet Ali Aydinlar University and Acibadem Labmed Health Services A.S., which also has yet to report the findings of its preclinical tests [@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].
+Following successful preclinical investigation [@doi:10.1073/pnas.2102775118], an intranasally administered deoptimized SARS-CoV-2 LAV known as COVI-VAC was developed by both New York-based Codagenix and the Serum Institute of India. 
+COVI-VAC entered phase I human trials and dosed its first participants in January 2021 [04619628][@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].  
+It is anticipated that the COVI-VAC phase I human trials will be complete by May 2022.
```
Comment (rando2) [inline content/22.vaccines.md:153]:
```suggestion
It is anticipated that the COVI-VAC phase I human trials will be completed by May 2022.
```

File: content/22.vaccines.md [hunk 11]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+Similarly,  Meissa vaccines in Kansas in the U.S., who also develop vaccines for respiratory syncytial virus (RSV), began enrollment for its phase I human trials on its intranasal live-attenuated chimeric vaccine candidate in March 2021, but they are still recruiting participants [@clinicaltrials:NCT04798001][@url:https://covid-19tracker.milkeninstitute.org/#vaccines_intro].
```
Comment (rando2) [inline content/22.vaccines.md:154]:
```suggestion
Similarly,  Meissa Vaccines in Kansas, U.S.A., which also develops vaccines for _Respiratory syncytial virus_ (RSV), began enrollment for phase I human trials on an intranasal live-attenuated chimeric vaccine candidate in March 2021 but is still recruiting participants [@clinicaltrials:NCT04798001; @url:https://covid-19tracker.milkeninstitute.org].
```
It switches back and forth between treating Meissa vaccines as "it" and "they" -- does this work?

File: content/22.vaccines.md [hunk 12]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+
+Despite the long and trusted history of LAV development, this vaccine strategy does not appear to be favoured for vaccine development against COVID-19.
```
Comment (rando2) [inline content/22.vaccines.md:156]:
```suggestion
Despite the long and trusted history of LAV development, this vaccine strategy does not appear to be favored for vaccine development against COVID-19.
```

File: content/22.vaccines.md [hunk 13]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
...
+Modern technologies such as mRNA vaccines and vectored vaccines seem to have been favoured due to their expediency and safety versus the time consuming nature of developing LAVs using a novel virus.
```
Comment (rando2) [inline content/22.vaccines.md:157]:
```suggestion
Modern technologies such as mRNA vaccines and vectored vaccines seem to have been favored due to their expediency and safety versus the time-consuming nature of developing LAVs using a novel virus.  
```

**Review by rando2 — APPROVED** [review]
This looks good to me @RLordan! Thank you for adding this! I made some edits, mostly just minor ones (sorry for Americanifying your spelling!)

**Comment by RLordan** [comment]
> This looks good to me @RLordan! Thank you for adding this! I made some edits, mostly just minor ones (sorry for Americanifying your spelling!)

All is forgiven :) thanks a million

File: content/22.vaccines.md [hunk 14]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
 
 ### Live-Attenuated Viruses
 
-<!--To Do: Not used this time I think?-->
-Although this is the vaccine development strategy with the longest history... (short paragraph explaining not really in play here)
+Live-attenuated vaccines (LAV) or replication-competent vaccines use a weakened living version of a disease-causing virus or a version of a virus that is similar to induce an immune response [@doi:10.1038/s41586-020-2798-3].
```
[the remark this answers is not in the document]
Comment (RLordan) [reply content/22.vaccines.md:126]: I think maybe you're right. I found this one hard to define.

File: content/22.vaccines.md [hunk 15]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
 
 ### Live-Attenuated Viruses
 
-<!--To Do: Not used this time I think?-->
-Although this is the vaccine development strategy with the longest history... (short paragraph explaining not really in play here)
+Live-attenuated vaccines (LAV) or replication-competent vaccines use a weakened living version of a disease-causing virus or a version of a virus that is similar to induce an immune response [@doi:10.1038/s41586-020-2798-3]. 
+The virus can be attenuated by passinging it in a foreign host, until as a consequence of selection pressure the virus loses its efficacy in the original host. 
+Alternatively, selective gene deletion or codon de-optimization can be utilized to attenuate the virus [@doi:10.1038/s41586-020-2798-3]. 
+LAVs are used globally to prevent diseases caused by viruses such as measles, rubella, polio, influenza, varicella zoster, and the yellow fever virus [@doi:10.1016/B978-0-323-35761-6.00002-X].
```
[the remark this answers is not in the document]
Comment (RLordan) [reply content/22.vaccines.md:129]: Actually all those listed are viruses as is..., I just shortened it to rubella instead of rubella virus, to reduce the length of the sentence. e.g. such as measles virus, rubella virus, polio virus, influenza virus, yellow fever virus, and those caused by Varicella zoster virus.

So should we italicize them all, or leave it as is...

File: content/22.vaccines.md (hunk 9 above)
Comment (RLordan) [reply content/22.vaccines.md:152]: Thanks for spotting!

File: content/22.vaccines.md (hunk 11 above)
Comment (RLordan) [reply content/22.vaccines.md:154]: maybe "...but they are still recruiting..." tricky one

File: content/22.vaccines.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (rando2) [reply content/22.vaccines.md:129]: Oops! I totally misread it. Let's just leave it how you wrote it and let the copy editors tell us if we're wrong!

File: content/22.vaccines.md (hunk 11 above)
Comment (rando2) [reply content/22.vaccines.md:154]: Maybe drop the subject entirely? 😆 
```suggestion
Similarly,  Meissa Vaccines in Kansas, U.S.A., which also develops vaccines for _Respiratory syncytial virus_ (RSV), began enrollment for phase I human trials on an intranasal live-attenuated chimeric vaccine candidate in March 2021 for which recruitment is ongoing [@clinicaltrials:NCT04798001; @url:https://covid-19tracker.milkeninstitute.org].
```

File: content/22.vaccines.md [hunk 16]
```diff
@@ -123,8 +123,38 @@ These diverse technology platforms include DNA, RNA, virus-like particle, recomb
 
 ### Live-Attenuated Viruses
 
-<!--To Do: Not used this time I think?-->
-Although this is the vaccine development strategy with the longest history... (short paragraph explaining not really in play here)
+Live-attenuated vaccines (LAV), or replication-competent vaccines, use a weakened living version of a disease-causing virus or a version of a virus that is modified to induce an immune response [@doi:10.1038/s41586-020-2798-3]. 
+The virus can be attenuated by passinging it in a foreign host until, as a consequence of selection pressure, the virus loses its efficacy in the original host.
```
Comment (rando2) [inline content/22.vaccines.md:127]:
```suggestion
The virus can be attenuated by passaging it in a foreign host until, as a consequence of selection pressure, the virus loses its efficacy in the original host. 
```

**Comment by rando2** [comment]
OK, I think this is ready to merge, as long as I didn't change the meaning with that last spelling suggestion? passinging vs passaging

File: content/22.vaccines.md (hunk 16 above)
Comment (RLordan) [reply content/22.vaccines.md:127]: Typo, my bad

**Comment by RLordan** [comment]
> OK, I think this is ready to merge, as long as I didn't change the meaning with that last spelling suggestion? passinging vs passaging

Passaging was correct, just a typo by me. All good here I think if you want to merge. 

Thanks @rando2

**Comment by rando2** [comment]
> Passaging was correct, just a typo by me. All good here I think if you want to merge.

Perfect! I assumed that's what happened but I've also learned a lot of new vocab over the last year and a half!
